helm/hpcc/values.schema.json

```diff
@@ -1865,8 +1865,8 @@
           "description": "Should the index node memory allocation flush the cache and retry if memory allocation fails"
         },
         "fieldTranslationEnabled": {
-          "default": "payload",
-          "enum": ["false", "true", "payload"],
```

Comment:
@jackdelv this change also needs to be in the previous build otherwise the option cannot be configured.

Reply:
Opened #18694